### PR TITLE
ghc 7.10.3b

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/ghc.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Ghc < Formula
   desc "Glorious Glasgow Haskell Compilation System"
   homepage "https://haskell.org/ghc/"
-  url "https://downloads.haskell.org/~ghc/7.10.3/ghc-7.10.3a-src.tar.bz2"
-  sha256 "877899988b64d13a86148f4db29b26d5adddef9207f718b726dc5c202d8efd8e"
+  url "https://downloads.haskell.org/~ghc/7.10.3/ghc-7.10.3b-src.tar.bz2"
```
Comment (DomT4) [inline Library/Formula/ghc.rb]: Is this stable? Can't find any reference to it on the upstream website.

Note the `a` release here doesn't signify alpha; the first 7.10.3 release they cut lacked some documentation so they tagged a new baby release. Alphabet releases aren't the norm.

Comment (apjanke) [inline Library/Formula/ghc.rb]: It's what the main upstream download page is pointing at now. https://www.haskell.org/ghc/download_ghc_7_10_3#sources

Referenced on mailing list here: http://osdir.com/ml/glasgow-haskell-users@haskell.org/2015-12/msg00011.html. Looks like it's just another issue with the release notes included in the distribution.

**Comment by apjanke** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Build fell apart on Yosemite:
```
utils/haddock/dist/build   -c utils/haddock/haddock-library/src/Documentation/Haddock/Utf8.hs -o utils/haddock/dist/build/Documentation/Haddock/Utf8.dyn_o 
<command line>: error: directory portion of "utils/haddock/dist/build/Haddock/GhcUtils.dyn_o" does not exist (used with "-o" option.)
<command line>: error: directory portion of "utils/haddock/dist/build/Documentation/Haddock/Utf8.dyn_o" does not exist (used with "-o" option.)
<command line>: error: directory portion of "utils/haddock/dist/build/Documentation/Haddock/Types.dyn_o" does not exist (used with "-o" option.)
make[1]: *** [utils/haddock/dist/build/Documentation/Haddock/Utf8.dyn_o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [utils/haddock/dist/build/Documentation/Haddock/Types.dyn_o] Error 1
make[1]: *** [utils/haddock/dist/build/Haddock/GhcUtils.dyn_o] Error 1
make: *** [all] Error 2
```

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/3d8d203c686c6b2fb54e75b2d574dc939bfba7bd. Thank you!

**Pull request closed without merging**